VideoApp/VideoApp/ViewControllers/Settings/SDKLogLevel/SDKLogLevelSettingsViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+class SDKLogLevelSettingsViewModel: SettingsViewModel {
+    let title = "SDK Log Levels"
+    var sections: [SettingsViewModelSection] {
+        [
+            .init(
+                rows: [
+                    .optionList(
+                        title: "Core",
```

Comment:
We use the string literals in a couple places. Is this something that should be coming from a constant of some sort? We use it here when setting up the option list and we set it in the individual view models. Would it make sense to reference a class variable from the ViewModel when setting up the titles?

Not earth shattering just a minor comment to reduce string literal duplication.

Reply:
Probably what makes sense is to use localized strings which would be sort of a built in way to avoid duplication, make future localization easier, and remove string literals from code. That might be easy to do.

Reply:
✅